setup.py

```diff
@@ -36,8 +36,8 @@
 
 # define install requirements
 install_requires_list = [
-        'numpy', 'pyyaml', 'scikit-learn', 'schema', 'py-cpuinfo', 'hyperopt', 'pandas', 'pycocotools',
-        'opencv-python', 'requests', 'psutil', 'Pillow', 'sigopt', 'prettytable', 'cryptography', 'Cython',
+        'numpy', 'pyyaml', 'scikit-learn', 'schema', 'py-cpuinfo', 'pandas', 'pycocotools',
+        'opencv-python', 'requests', 'psutil', 'Pillow', 'prettytable', 'cryptography', 'Cython',
```

Comment:
cython should be removed. pls confirm with suyue

Reply:
Hi Feng, have confirmed with @chensuyue that she will remove `Cpython` by another PR.